service/src/main/java/bio/terra/externalcreds/controllers/OidcApiController.java

```diff
@@ -50,47 +50,39 @@ public ResponseEntity<List<String>> listProviders() {
   }
 
   @Override
-  public ResponseEntity<LinkInfo> getLink(Provider providerName) {
+  public ResponseEntity<LinkInfo> getLink(Provider provider) {
     var samUser = samUserFactory.from(request);
-    var linkedAccount =
-        linkedAccountService.getLinkedAccount(samUser.getSubjectId(), providerName.toString());
+    var linkedAccount = linkedAccountService.getLinkedAccount(samUser.getSubjectId(), provider);
     return ResponseEntity.of(linkedAccount.map(OpenApiConverters.Output::convert));
   }
 
   @Override
-  public ResponseEntity<String> getAuthUrl(Provider providerName, String redirectUri) {
+  public ResponseEntity<String> getAuthUrl(Provider provider, String redirectUri) {
```

Comment:
Is this route still being used in the Terra UI? (Same with the createLink route) It would be nice to delete that either in this PR or a follow-on! 🔪

Reply:
Yeah, its still using this. I can fold that work into this ticket, I think.

Reply:
Hmm if it's still being used, I don't think removing those routes needs to block this PR from being merged!